docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: "Introducing the TestCafe Jenkins plugin"
+permalink: /media/team-blog/:title.html
+isTeamBlog: true
+author: Sergey Shurygin, Eugene Titerman
+---
+# Introducing the TestCafe Jenkins plugin
+
+TestCafe offers a number of debugging features, including the capability to automatically capture videos and screenshots during the testing process. You can record all your tests or only those that failed, generate multiple video files or stitch them together, and even single out specific interactions (see [the documentation](https://devexpress.github.io/testcafe/documentation/guides/advanced-guides/screenshots-and-videos.html) for the full list of available options).
+
+But automating this process was not always easy. For a long time, Jenkins users had to manually match completed tests to their artifacts. The recent introduction of the [TestCafe Jenkins plugin](https://plugins.jenkins.io/testcafe/) changes that. The screenshots and videos taken during the test can now be automatically linked to on the Jenkins test results page. You don't even need to keep your testing server running — all the necessary files are stored right inside the Jenkins build folder.
+
+<!--more-->
+
+**It looks something like this:**
```

Comment:
Do we really need this emphasis (I mean bold font)? 

I would also go with a bit less colloquial phrase here:
`Your test results page will look like this:`

There are two reasons: 'it' is too vague here (what does it refer to?), and 'something' is too uncertain for a corporate blog (we should know for sure what our plugin renders).